web/src/main/java/no/nav/modiapersonoversikt/rest/person/pdl/PersondataFletter.kt

```diff
@@ -0,0 +1,547 @@
+package no.nav.modiapersonoversikt.rest.person.pdl
+
+import no.nav.modiapersonoversikt.legacy.api.domain.norg.AnsattEnhet
+import no.nav.modiapersonoversikt.legacy.api.domain.pdl.generated.HentPersondata
+import no.nav.modiapersonoversikt.legacy.api.domain.pdl.generated.HentPersondata.AdressebeskyttelseGradering.*
+import no.nav.modiapersonoversikt.legacy.api.domain.pdl.generated.HentPersondata.KontaktinformasjonForDoedsboSkifteform.ANNET
+import no.nav.modiapersonoversikt.legacy.api.domain.pdl.generated.HentPersondata.KontaktinformasjonForDoedsboSkifteform.OFFENTLIG
+import no.nav.modiapersonoversikt.legacy.api.utils.TjenestekallLogger
+import no.nav.modiapersonoversikt.service.dkif.Dkif
+import no.nav.modiapersonoversikt.service.enhetligkodeverk.EnhetligKodeverk
+import no.nav.tjeneste.virksomhet.person.v3.informasjon.BankkontoNorge
+import no.nav.tjeneste.virksomhet.person.v3.informasjon.BankkontoUtland
+import no.nav.tjeneste.virksomhet.person.v3.informasjon.Bruker
+import no.nav.tjeneste.virksomhet.person.v3.meldinger.HentPersonResponse
+import org.slf4j.LoggerFactory
+import java.time.LocalDate
+import no.nav.modiapersonoversikt.service.enhetligkodeverk.KodeverkConfig as Kodeverk
+
+class PersondataFletter(val kodeverk: EnhetligKodeverk.Service) {
+    private val log = LoggerFactory.getLogger(PersondataFletter::class.java)
```

Comment:
Kan ikke se at vi bruker loggeren per nå, men det er kanskje gjort klart for senere?

Reply:
Slettet den